core/src/actions/properties.ts

```diff
@@ -185,6 +187,7 @@ export class PropertyEdit extends AuthenticatedAction {
       sourceId: { required: false },
       options: { required: false },
       filters: { required: false },
+      keepValueIfNotFound: { required: false },
```

Comment:
Doesn't look like this new input/param is used in the action?

Reply:
It is! the action uses `await property.update(params);`, and `keepValueIfNotFound` is in that params obj (along with other keys)